### DISCUSSION TITLE
extension is not working for me for some reason

### DISCUSSION THREAD

**Discussion by lazizdevv** [tailwindlabs/tailwindcss#13894] [Help]
![Screenshot 2024-06-25 183436](https://github.com/tailwindlabs/tailwindcss/assets/148150447/42476809-cc9d-4fae-a598-e8b422a27033)

please help me

**Comment by evert-vd** [comment]
Han publicado la versión v0.12.1 de tailwindcss-intellisense que soluciona el error.
https://github.com/tailwindlabs/tailwindcss-intellisense/issues/988
